forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/FRSessionTest.java

```diff
@@ -7,6 +7,9 @@
 
 package org.forgerock.android.auth;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Revert change or update copyright

Reply:
Reverted the changes and left the original copyright note...